addons/im_livechat/static/src/core/common/chat_window_model_patch.js

```diff
@@ -40,6 +40,10 @@ const chatWindowPatch = {
                 }
                 this.actionsDisabled = true;
                 this.livechatStep = CW_LIVECHAT_STEP.CONFIRM_CLOSE;
+                if (!isSelfVisitor && this.thread.channel_member_ids.length > 2) {
```

Comment:
Note that line 54 is equivalent to `isSelfVisitor`. Could be replace to avoid typos with the long line and `?.` cautiously put there.